crates/context/src/lib.rs

```diff
@@ -307,11 +307,18 @@ impl ContextManager {
         blob_id: calimero_primitives::blobs::BlobId,
         source: calimero_primitives::application::ApplicationSource,
         version: Option<semver::Version>,
+        contract_app_id: Option<String>,
     ) -> eyre::Result<calimero_primitives::application::ApplicationId> {
+        let boxed_id: Box<str> = match contract_app_id {
```

Comment:
what is boxed_id? please use name by meaning, not by type

Reply:
Updated